src/i18n/en.js

```diff
@@ -63,18 +63,34 @@ const messages_en = {
     add_permission_google_account: `Add permission for Google accounts.`,
     free_alt_image: `big query`,
     free_btn_redirect: `Get to know our Plus plan`,
-    free_text_data_studio_MD: `Learn to configure our dashboard [Data Studio in a few steps](https://datastudio.google.com/overview),
+    free_text_data_studio_MD: `[Configure a predesigned dashboard in a few steps](https://datastudio.google.com/overview) and get Doppler’s reports in Google Data Studio.,
     or access BigQuery through another data visualization tool such as Tableau.`,
-    free_text_strong: `¡Power up your account with BigQuery reports!`,
-    free_text_summary: `Integrate Google BigQuery to analyze detailed reports on your campaigns, your subscribers and other data of your account in a customizable dashboard`,
-    free_title: `Big Query`,
-    free_ul_item_filter: `You can filter the reports based on all your campaigns or select only some in particular`,
-    free_ul_item_insights: `Get valuable insights about your account`,
-    plus_button_add: `+`,
-    plus_button_remove: `-`,
-    plus_error_message_email_exists: `The email already exists in configuration`,
-    plus_loading_message: `Loading ...`,
+    free_text_strong: `Boost your Campaigns with reports in Google Data Studio, Tableau, Power BI and other Data Analytics tools.`,
+    free_text_summary: `Get detailed reports to analyze the performance of your Email & Automation Marketing Campaigns. With this integration you can:`,
+    free_title: `Connect Doppler with BigQuery`,
+    free_ul_item_filter: `Apply different filters to focus the analysis on one or more strategy's issues.`,
+    free_ul_item_insights: `Work from a customizable dashboard with editable views.`,
+    free_ul_item_strategy: `Get valuable information for your strategy.`,
+    plus_beginning: `Start`,
+    plus_big_query: `BigQuery`,
+    plus_body_step_one_MD: `Give access to BigQuery. [HELP] (https://cloud.google.com/bigquery/docs/?hl=es-419)`,
+    plus_body_step_two_MD: `Generate Doppler reports at Google Data Studio. [HELP](https://datastudio.google.com/overview)`,
+    plus_configuration: `Configuration`,
+    plus_control_panel: `Control Panel`,
+    plus_header_MD: `Get detailed reports to analyze the performance of your Email & Automation Marketing Campaigns.`,
+    plus_input_place_holder: `Add google account.`,
+    plus_message_remember: `Remember to Save Changes.`,
```

Comment:
```suggestion
    plus_message_remember: `Remember to save changes.`,
```
We use only capitals for Doppler products.

Reply:
@jmonacchi can you recheck these corrections? 🙏